var/spack/repos/builtin/packages/freetype/package.py

```diff
@@ -30,7 +32,6 @@ class Freetype(AutotoolsPackage):
 
     depends_on("bzip2")
     depends_on("libpng")
-    depends_on("pkgconfig", type="build")
```

Comment:
Why is this not required anymore?

Reply:
This will not be required as we are not longer using autotools.

Reply:
CMake still seems to use pkg-config, though: https://github.com/freetype/freetype/blob/master/CMakeLists.txt#L251

Reply:
It's certainly not a hard requirement as I am able to build perfectly fine without it on both Unix and Windows, and as evidenced by the lack of `REQUIRED` in the find package invocation. Many find package modules may use PkgConfig to perform some package detection, but it's not required, and the only direct use I see is the `pkg_check_modules` which comes from the CMake module so it'll be available anyway. I'll make the requirement optional and off on Windows and on everywhere else, I think that's probably best, as you're right, other platforms may get some utility out of it (although I'm not sure how much)

Reply:
Perfect, thanks!